src/style.css

```diff
@@ -0,0 +1,85 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: "Avenir-Book";
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.form {
+  margin: 20px 0;
+}
+
+.input {
+  width: 100%;
+  height: 70px;
+  padding-left: 62px;
+  display: inline-block;
+
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+
+  font-family: Avenir-Heavy, sans-serif;
```

Comment:
add font-family for html tag and here you can use `inherit`
https://developer.mozilla.org/en-US/docs/Web/CSS/inherit